pages/common/kcat.md

```diff
@@ -0,0 +1,36 @@
+# kcat
+
+> Apache Kafka producer and consumer tool.
+> More information: <https://github.com/edenhill/kcat>.
+
+- Consume messages starting with the newest offset:
+
+`kcat -C -t {{topic}} -b {{brokers}}`
+
+- Consume messages starting with the oldest offset and exit after the last message is received:
+
+`kcat -C -t {{topic}} -b {{brokers}} -o beginning -e`
+
+- Consume messages as a Kafka consumer group:
+
+`kcat -G {{group_id}} {{topic}} -b {{brokers}}`
```

Comment:
Does it miss `-t` here?
```suggestion
`kcat -G {{group_id}} -t {{topic}} -b {{brokers}}`
```

Reply:
@blueskyson you are right, will add the missing `-t` and update the PR